packages/lib-classifier/src/store/TutorialStore/TutorialStore.js

```diff
@@ -65,8 +65,14 @@ const TutorialStore = types
     },
 
     get hasNotSeenTutorialBefore () {
+      const uppStore = tryReference(() => getRoot(self).userProjectPreferences)
```

Comment:
`tryReference` can only be used with `types.safeReference`, I think.

Reply:
```suggestion
      const uppStore = getRoot(self).userProjectPreferences)
```
maybe should be just ^ for now?